cg/cli/workflow/rnafusion/base.py

```diff
@@ -129,7 +129,7 @@ def run(
             case_id=case_id,
             log=log,
             work_dir=work_dir,
-            resume=resume,
+            resume=not from_start,
```

Comment:
```suggestion
            resume=not_from_start,
```
I assume? or perhaps just `resume`

Reply:
well, I wanted to get the opposite of `from_start`. Meaning that if running with `from_start` set as True (so, starting from the beginning without resuming), then resume should be False (so the opposite), and the other way around. 

I could indicate it more clearly by adding:

```
    resume = True
    if from_start:
        resume = False
```
or
```
    resume = not from_start
```

and then just leaving:
```suggestion
            resume=resume,
```

Let me know what you prefer @henrikstranneheim

Reply:
I think this looks clear.

```
 resume = True
    if from_start:
        resume = False
```

or even shorter:
```
 resume = False if from_start else True
```

Reply:
🤩